lib/management/configuration.js

```diff
@@ -171,7 +171,7 @@ function patchConfiguration(newConf, log, cb) {
                 _config.setLocationConstraints(locations);
             } catch (error) {
                 log.info('could not apply configuration version location ' +
-                    'constraints', { error });
+                    'constraints', { error: error.message });
```

Comment:
There may be something wrong with the exception thrown by `_config.setLocationConstraints()` so I would look at that instead.

We usually log actual error objects as the `error` attribute so the code is fine as it is. The `error` attribute of log lines actually has to be an object and not a string (we use the `errorMessage` attribute for error messages) otherwise any log indexer will reject the line because the implicit schema and the line don't match.

Reply:
As this logging statement is inside a try-catch, you must explicitly build the logging object.